Github/legos/github_assign_issue/README.md

```diff
@@ -0,0 +1,29 @@
+[<img align="left" src="https://unskript.com/assets/favicon.png" width="100" height="100" style="padding-right: 5px">](https://unskript.com/assets/favicon.png) 
+<h2>Github Assign Issue</h2>
+
+<br>
+
+## Description
+This Lego assigns a Github issue to a user
+
+## Lego Details
+
+    github_assign_issue(handle, owner:str, repository:str, issue_number:int, assignees:str)
```

Comment:
why is it assignees, should be just assignee, same thing below

Reply:
Changed